pages/linux/uuidparse.md

```diff
@@ -0,0 +1,28 @@
+# uuidparse
+
+> Parse universally unique identifiers.
```

Comment:
```suggestion
> Parse universally unique identifiers.
> See also: `uuidgen`.
```
We don't have this command documented yet, but we can document it later.

Reply:
A page for uuidgen already exists